packages/expo-blur/src/BlurView.web.tsx

```diff
@@ -1,28 +1,51 @@
 import PropTypes from 'prop-types';
 import * as React from 'react';
-import { View, StyleSheet, ViewPropTypes } from 'react-native';
+import { View, ViewPropTypes } from 'react-native';
 
-type Props = {
-  tint: BlurTint;
-} & React.ComponentProps<typeof View>;
-type BlurTint = 'light' | 'dark' | 'default';
+import { BlurTint, Props } from './BlurView.types';
+import getBackgroundColor from './getBackgroundColor';
 
+function isBlurSupported(): boolean {
```

Comment:
Move helper functions below the functions/classes that use them --> 

1. code of primary interest
2. implementation details

establishes context better than the other way around